lite2/client.go

```diff
@@ -764,6 +774,11 @@ func (c *Client) bisection(
 			c.trustLevel)
 		switch err.(type) {
 		case nil:
+			// termination clause
+			if interimHeader.Height == newHeader.Height {
```

Comment:
why it's better than for trustedHeader.Height < newHeader.Height { ?

Reply:
it saves having to finish the rest of the loop before termination